cmd/process-agent/command/main_common.go

```diff
@@ -267,12 +259,7 @@ func initMisc(deps miscDeps) error {
 	deps.Lc.Append(fx.Hook{
 		OnStart: func(startCtx context.Context) error {
 
-			err := tagger.Init(startCtx)
-			if err != nil {
-				log.Errorf("failed to start the tagger: %s", err)
-			}
-
-			err = manager.ConfigureAutoExit(startCtx, deps.Config)
+			err := manager.ConfigureAutoExit(startCtx, deps.Config)
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** Are you sure want to pass `startCtx` and not `appCtx` as first parameter to `ConfigureAutoExit(...)` call ?  Because this context provided by fx only live for a given time (15 secs by default). In this case it would say that the *autoexit* behaviour will stop within 15 seconds.

Here‏ is the fx specification about `onStart` context:
```
		// To mitigate the impact of deadlocks in application startup and
		// shutdown, Fx imposes a time limit on OnStart and OnStop hooks. By
		// default, hooks have a total of 15 seconds to complete. Timeouts are
		// passed via Go's usual context.Context.
```
link: https://pkg.go.dev/go.uber.org/fx#section-readme